docs-source/content/security/openshift.md

```diff
@@ -44,3 +44,8 @@ spec:
 For additional information about OpenShift requirements and the operator,
 see the [OpenShift]({{<relref  "/userguide/introduction/introduction.md#openshift">}}) section in the User Guide.
 {{% /notice %}}
+
+#### Using a dedicated namespace
+
+When the user that installs an individual instance of the operator does not have the required privileges to create resources at the Kubernetes cluster level, a dedicated namespace can be used for the operator instance and all WebLogic domains that it manages. For more details of the `dedicated` setting, please refer to [Operator Helm configuration values]({{< relref "/userguide/managing-operators/using-the-operator/using-helm.md#operator-helm-configuration-values" >}}).
```

Comment:
and all WebLogic -> and all the WebLogic
For more details of the -> For more details about the

Reply:
done